examples/ai-billing/app/api/openai/route.ts

```diff
@@ -0,0 +1,98 @@
+import { auth } from "@/auth";
+import { Unkey, verifyKey } from "@unkey/api";
+import { cookies } from "next/headers";
+import { NextResponse } from "next/server";
+import OpenAI from "openai";
+
+// Create an OpenAI API client (that's edge friendly!)
+const openai = new OpenAI({
+  apiKey: process.env.OPENAI_API_KEY,
+});
+
+// Set the runtime to edge for best performance
+export const runtime = "edge";
+
+export async function POST(req: Request, _res: Response) {
```

Comment:
@domeccleston what do you think of refactoring the cookie logic in a utility function outside of this file?

right now it looks quite intimidating and like a lot of work to do just to use unkey

Reply:
Done